frontend/src/scenes/setup/PreflightCheck.js

```diff
@@ -0,0 +1,236 @@
+                                style={{ position: 'absolute', right: 16 }}
+                                data-attr="preflight-refresh"
+                                icon={<SyncOutlined />}
+                                onClick={() => window.location.reload()}
+                            >
+                                Refresh
+                            </Button>
+                        </Row>
+                        <div>
+                            Tell us what you plan to do with this installation to make sure your infrastructure is ready
+                        </div>
+                        <div
+                            className="text-center"
+                            style={{ padding: 32, display: 'flex', justifyContent: 'center', maxWidth: 600 }}
+                        >
+                            {!state.mode && (
+                                <>
+                                    <Button
+                                        type="default"
+                                        data-attr="preflight-experimentation"
+                                        onClick={() => handleModeChange('Experimentation')}
+                                    >
+                                        🧪 Just experimenting
+                                    </Button>
+                                    <Button
+                                        type="primary"
+                                        style={{ marginLeft: 16 }}
+                                        data-attr="preflight-live"
+                                        onClick={() => handleModeChange('Live')}
+                                    >
+                                        🚀 Live implementation
+                                    </Button>
+                                </>
+                            )}
+
+                            {state.mode && (
+                                <>
+                                    <Row>
+                                        {checks.map((item) => (
+                                            <PreflightItem key={item.id} {...item} />
+                                        ))}
+                                    </Row>
+                                </>
+                            )}
+                        </div>
+
+                        <div style={{ fontSize: 12 }}>
+                            We will not enforce some security requirements on experimentation mode.
+                        </div>
+                    </Card>
+                    <div className="space-top text-center">
+                        {!isReady && <b>Please review the items above before continuing</b>}
+                        {isReady && (
+                            <b style={{ color: '#28A745' }}>
+                                Congratulations! Looks like your infrastructure is ready!
+                            </b>
+                        )}
+                    </div>
```

Comment:
![Continue](https://user-images.githubusercontent.com/4550621/90758422-4c112d80-e2df-11ea-8eb2-8ce6409b541b.png)
This may distract from the actual preflight flow. May possibly be better to have it appear on the last step if all went well.